bemani/backend/iidx/cannonballers.py

```diff
@@ -290,6 +298,17 @@ def game_to_db_rank(self, game_dan: int, cltype: int) -> int:
         else:
             raise Exception('Invalid cltype!')
 
+    def game_to_db_chart(self, db_chart: int) -> int:
```

Comment:
No need for db_to_game_chart in 25?

Reply:
There's no `db_to_game_chart` because that is technically handled by `make_score_struct` in base.py. Unfortunately. `make_score_struct` will either likely need to be refactored or outright overloaded in heroic verse and later because the struct is changed. Currently I just overload the function but that feels wasteful...

Reply:
I need to have a look at that but I only have a few minutes so I'll peek and ponder later.

Reply:
Ah, yeah. That and `make_beginner_struct` probably shouldn't go into base.py. That's my bad. That function is definitely specific to the parts of the series that use it, which admittedly is most of them. However, maybe they need to move to a common.py which subclasses from base.py or something, and then we can use it as a mixin to the older versions? Dunno, that's probably good enough and then do something different and custom for HV since its no longer a common function. Don't think any of that has to happen here though. Thoughts?